client/src/components/NewNavbar.jsx

```diff
@@ -46,7 +46,7 @@ export default function NewNavbar({ position }) {
   const navLinksDashboard = [
     {
       title: "account",
```

Comment:
https://github.com/digitomize/digitomize/issues/764#issuecomment-2100442322

You didn't changed the title and the redirection is also not quite right!